soroban-sdk/src/env.rs

```diff
@@ -469,6 +469,79 @@ use std::{path::Path, rc::Rc};
 #[cfg(any(test, feature = "testutils"))]
 use xdr::{LedgerEntry, LedgerKey, LedgerKeyContractData, SorobanAuthorizationEntry};
 
+#[cfg(any(test, feature = "testutils"))]
+pub struct TestStellarAssetContractUtil {
+    pub address: Address,
+    env: Env,
+    issuer_id: xdr::AccountId,
+}
+
+#[cfg(any(test, feature = "testutils"))]
+impl TestStellarAssetContractUtil {
+    pub fn get_issuer_flags(&self) -> u32 {
```

Comment:
The prefix `get_` is not preferred on functions that return the value named by the function. See the Rust naming conventions for examples:
- https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Removing get makes this function more consistent with existing APIs in the SDK:
- https://docs.rs/soroban-sdk/latest/soroban_sdk/ledger/struct.Ledger.html

```suggestion
    pub fn flags(&self) -> u32 {
```

Reply:
Done